logging/logs.rb

```diff
@@ -0,0 +1,19 @@
+#Types of logs:
+#   log.debug "a very nice little debug message"
+#   log.info "things are operating nominally"
+#   log.warn "this is your last warning"
+#   log.error StandardError.new("something went horribly wrong")
+#   log.fatal "I Die!"
+
+require 'logging'
+require 'environment.rb'
+
+# Creates a logfile with the level 'debug'
+#   that appends to log_file.log
+log = Logging.logger['log_file']
```

Comment:
In the beginning, I wasn't sure where to put it so I created a folder for it. I figured that if I needed to move it elsewhere, I could do so later.